main.py

```diff
@@ -0,0 +1,31 @@
+class Hotel():
+    static = 0
+
+    def __init__(self, name='', visitor_of_year=0, number_of_rooms=0):
+        self.name = name
+        self.visitor = visitor_of_year
+        self.number = number_of_rooms
+
+    def __str__(self):
+        return f"name:{self.name} visitor:{self.visitor} number:{self.number}"
+
+    def __del__(self):
+        print("викликаэмо деструктор")
+
+    @staticmethod
+    def staticmethod(static):
+        return f"static={static}"
+
+
+Lisha_1=Hotel("Lisha_1",10,10)
+print(Lisha_1.__str__())
```

Comment:
Слідкуємо за PEP8.
Виклик `__str__()` не потрібен, `print` викликає `__str__()` неявно.

```suggestion
lisha_1 = Hotel("Lisha_1",10,10)
print(lisha_1)
```